code/src/main/java/com/onlinecontacttracing/servlets/authentication/AuthenticationScope.java

```diff
@@ -0,0 +1,33 @@
+package com.onlinecontacttracing.authentication;
+
+import com.google.api.services.calendar.CalendarScopes;
+import com.google.api.services.people.v1.PeopleServiceScopes;
+
+public enum AuthenticationScope {
+  CALENDAR(CalendarScopes.CALENDAR_READONLY), CONTACTS(PeopleServiceScopes.CONTACTS_READONLY);
+
+  private String scopeName;
+  AuthenticationScope(String scopeName) {
+    this.scopeName = scopeName;
+  }
+
+  public static String getScopeName(AuthenticationScope scope) {
+    for (AuthenticationScope authenticationScope : AuthenticationScope.values()) { 
+      if (authenticationScope.equals(scope)) {
+        return scope.scopeName;
+      }
```

Comment:
this can actuall all just be replaced by an instance method!

```
public String getScopeName() {
  return this.scopeName;
}
```

Reply:
done